Gems/Atom/Tools/AtomToolsFramework/Code/Source/Document/AtomToolsDocumentMainWindow.cpp

```diff
@@ -500,4 +500,4 @@ namespace AtomToolsFramework
     }
 } // namespace AtomToolsFramework
 
-//#include <Document/moc_AtomToolsDocumentMainWindow.cpp>
+//#include <AtomToolsFramework/Document/moc_AtomToolsDocumentMainWindow.cpp>
```

Comment:
I've seen a few of these commented out moc includes, are they necessary?

Reply:
I need to look into why these are not being generated. Probably the wrong include path or nothing to generate for the derived classes.